config/packages/doctrine.yaml

```
@@ -10,5 +10,11 @@ doctrine:
         driver: 'pdo_mysql'
         server_version: '5.7'
         charset: UTF8
-
         url: '%env(resolve:DATABASE_URL)%'
+    orm:
+        dql:
+            string_functions:
+                DAY: DoctrineExtensions\Query\Mysql\Day
+                HOUR: DoctrineExtensions\Query\Mysql\Hour
```

Why do we need this precision in the provider? 

Also, can we put them in some order?
```suggestion
                HOUR: DoctrineExtensions\Query\Mysql\Hour
                DAY: DoctrineExtensions\Query\Mysql\Day
```

---

because our new graph proposed by @kulczy needs data aggregates by these intervals,
And also I have put it in alphabetical order (your suggestion was my second choice) - I am not sure what version is better :D